ASUWebPlatforms/webspark-ci
# WS2-1860 - No Alpha filter on Faculty Rank

### Description

Update logic on Web Dir admin UI to not show "alpha filter" option for Faculty Rank, since it's not approved for that Web Directory variation. While at it, I fixed a bug related to the Titles Filter group not honoring conditional logic - I removed it and surfaced up the field inside which does correctly show/hide.

Also updated to latest `app-webdir-ui` component.

###  Testing
1. View the forms for various Web Directory listing types - the "Disable Alphabetical filter" option should be hidden on the Faculty Rank listing type, and visible on all others.
2. View the listings associated with those types - the alpha listing setting should be honored on all variations except the Faculty Rank listings where it should never display.
3. Verify the title Filter is no longer in a collapsible fieldset.

### Links

- [JIRA ticket](https://asudev.jira.com/browse/WS2-1860)

### Checklist
